fitlins/cli/run.py

```diff
@@ -77,8 +77,8 @@ def get_parser():
                              'removed)')
     g_bids.add_argument('-m', '--model', action='store', default='model.json',
                         help='location of BIDS model description (default bids_dir/model.json)')
-    g_bids.add_argument('-p', '--preproc-dir', action='store', default='fmriprep',
-                        help='location of preprocessed data (default output_dir/fmriprep)')
+    g_bids.add_argument('-p', '--preproc-dir', action='store', default=None,
+                        help='location of preprocessed data (default bids_dir/fmriprep)')
```

Comment:
`bids_dir/derivatives/fmriprep`?

I take it there's a case where you want your outputs in a different derivatives directory?

Reply:
No, I've mentioned this before, but `bids_dir/derivatives/fmriprep` is a much more sensible default for me. That's where I would think 90% of people would place their fmriprep directory. Doesn't make sense for it to be in the "output" folder for me (especially if people are not putting the fitlins outputs in derivatives)

But maybe the description should be changed. if preproc_dir = None, and only ` bids_dir` is passed to `BIDSLayout`, all derivatives will be recursively indexed. So if there's an fmriprep folder in there, the preproc files will be available by default.

Reply:
The output directory is a derivatives directory, kind of by definition, since we put the fitlins derivatives in it. Similarly, with fmriprep, that's where we look for FreeSurfer derivatives. It seems much more common to me to keep your derivatives separate from your raw data than to have some of your derivatives in `bids_root/derivatives` and some out of it.

But this may be a false impression. What's your use case where you aren't putting your fitlins outputs next to you fmriprep outputs?

Reply:
I'm okay with leaving this as you have it for right now. Before the next release we should decide whether your approach or mine is the long-term one. I don't want to change CLI defaults too often.

Reply:
My derivatives are going in `bids_root/derivatives` but a bit deeper than that. E.g.: `bids_root/derivatives/neuroscout/analysis_number/`

It was my impression that most ppl would keep all derivatives w/ their raw data, but maybe not... worth polling and asking around a bit. 

I could imagine ppl might treat `fitlins` outputs a bit differently than your average derivative output since it will be run many times, for many models. Whereas you might only run fmriprep and freesurfer once.